test/shared/src/test/scala/monocle/std/StringsSpec.scala

```diff
@@ -1,11 +1,29 @@
 package monocle.std
 
+import java.util.UUID
+
 import monocle.MonocleSuite
 import monocle.function.Plated._
 import monocle.law.discipline.function._
 import monocle.law.discipline.{IsoTests, PrismTests, TraversalTests}
+import org.scalacheck.Arbitrary
+
+import scalaz.Equal
 
 class StringsSpec extends MonocleSuite {
+  implicit def arbitraryUUID: Arbitrary[UUID] = Arbitrary(UUID.randomUUID)
+
+  implicit def equalUUID: Equal[UUID] = new Equal[UUID] {
```

Comment:
You can use `equalA` https://github.com/scalaz/scalaz/blob/series/7.3.x/core/src/main/scala/scalaz/Equal.scala#L44